configure

```diff
@@ -973,7 +973,7 @@ case "$target" in
   powerpc*-*-linux*)            arch=power;
                                 if $arch64; then model=ppc64; else model=ppc; fi
                                 system=elf;;
-  powerpc-*-netbsd*)            arch=power; model=ppc; system=elf;;
+  powerpc-*-netbsd*)            arch=power; model=ppc; system=bsd_elf;;
```

Comment:
This was set to `elf` (originally it was `bsd`) to fix [MPR#4116](https://caml.inria.fr/mantis/view.php?id=4116). Any idea of why the setting needs to change now?

Reply:
Because with `system=elf`, we fall into the
    power,gcc*,elf,ppc)   partialld="ld -r -m elf32ppclinux";;
which we don't want to.